src/main/java/no/stelar7/api/r4j/basic/calling/DataCallBuilder.java

```diff
@@ -288,12 +288,27 @@ private String postProcessDDragonMany(String returnValue)
     
     private String postProcessDDragonAddId(String returnValue)
     {
+        List<String> buggedElem = new ArrayList<>();
         JsonObject elem   = (JsonObject) JsonParser.parseString(returnValue);
         JsonObject parent = elem.getAsJsonObject("data");
         for (String key : new HashSet<>(parent.keySet()))
         {
             JsonObject child = parent.getAsJsonObject(key);
-            child.addProperty("id", key);
+            try 
+            {
+                int keyAsInt = Integer.parseInt(key);
+                child.addProperty("id", keyAsInt);
+                
+            }   catch(NumberFormatException e) 
+            {
+                buggedElem.add(key);
```

Comment:
Do `parent.remove(key)` instead, that way you dont need to have the `buggedElem` list.

Thanks for the PR!

Reply:
Way better! Ty